systemtest/src/main/java/hivemall/systemtest/ConvertToMsgpack.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Hivemall: Hive scalable Machine Learning Library
+ *
+ * Copyright (C) 2016 Makoto YUI
+ * Copyright (C) 2013-2015 National Institute of Advanced Industrial Science and Technology (AIST)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package hivemall.systemtest;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.msgpack.core.MessagePack;
+import org.msgpack.core.MessagePacker;
+import org.msgpack.value.ValueFactory;
+
+import javax.annotation.Nonnull;
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.util.List;
+import java.util.zip.GZIPOutputStream;
+
+public class ConvertToMsgpack {
+    @Nonnull
+    private final File file;
+    @Nonnull
+    private final List<String> header;
+    @Nonnull
+    private final CSVFormat format;
+
+
+    public ConvertToMsgpack(File file, List<String> header, CSVFormat format) {
```

Comment:
add `@Nonnull` to arguments if it is guaranteed.
If not, use `@CheckForNull` for arguments and then final member instances should be `@Nonnull` .